File: sdks/php-opencloud/challenges/authenticate.php
<?php
require 'vendor/autoload.php';
use OpenCloud\Rackspace;

$endpoint = getenv('RAX_AUTH_URL') . '/v2.0/';
$credentials = array(
    'username' => getenv('RAX_USERNAME'),
    'apiKey' => getenv('RAX_API_KEY')
);

$rackspace = new Rackspace($endpoint, $credentials);
$rackspace->Authenticate();
echo('Authenticated')
?>

File: sdks/openstack.net/challenges/Authenticate.cs
using System;
using net.openstack.Providers.Rackspace;
using net.openstack.Core.Providers;
using net.openstack.Core.Exceptions.Response;
using net.openstack.Providers.Rackspace.Objects;

namespace openstack.net
{
	class Authenticate : Challenge
	{
		public int Run (string[] args)
		{
			IIdentityProvider identityProvider = new CloudIdentityProvider ();
			var userAccess = identityProvider.Authenticate (new RackspaceCloudIdentity {
				Username = Environment.GetEnvironmentVariable("RAX_USERNAME"),
				APIKey = Environment.GetEnvironmentVariable("RAX_API_KEY")
			});
			Console.WriteLine ("Authenticated!");
			return 0;
		}
	}
}
